deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js

```diff
@@ -315,21 +315,29 @@ Vue.component("placements-select", {
     <label class="deployToolTip control-label col-xs-2" data-toggle="tooltip" v-bind:title="title">\
       {{label}}\
     </label>\
-    <div class="col-xs-7"><div v-bind:class="groupStyle">\
+    <div class="col-xs-6"><div v-bind:class="groupStyle">\
           <select class="form-control chosen-select"  v-on:change="updateValue($event.target.value)" multiple>\
             <option v-for="option in selectoptions" v-bind:value="option.value" v-bind:selected="option.isSelected">{{option.text}}</option>\
           </select>\
     <span v-if="showhelp" class="input-group-btn">\
     <button class="deployToolTip btn btn-default" type="button" data-toggle="tooltip" v-on:click="helpClick" title="click to see more information">\
         <span class="glyphicon glyphicon-question-sign"></span>\
     </button>\
-   </span></div>\
+    </span></div>\
     </div>\
-    <div class="col-xs-3">\
-      <input type="checkbox" id="checkbox" v-bind:checked="assignpublicip" v-on:click="assignipchange($event.target.checked)">\
-      <label for="checkbox">Assign Public IP</label>\
-    <div></div>',
-    props: ['label', 'title', 'selectoptions', 'showhelp', 'assignpublicip'],
+    <div class="col-xs-2" v-if="showsubnettype">\
+        <input type="radio" id="public" value="public" v-model="subnettype" v-on:click="filterclick($event.target.value)">\
+        <label for="public">Public subnets</label>\
+        <br>\
+        <input type="radio" id="private" value="private" v-model="subnettype" v-on:click="filterclick($event.target.value)">\
```

Comment:
nit: ID is unique on the page right? It might be too general if you just call it "public" and "private".